modules.d/70kernel-modules-export/copymods.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+command -v getarg > /dev/null || . /lib/dracut-lib.sh
+
+copymods=$(getarg rd.driver.export=)
```

Comment:
Since we settled on the name, we can drop the `copy` everywhere. I would rename `copymods` to `driver_export` and `copymods.sh` to `modules-export.sh`.